modules/oe_content_event/tests/Kernel/EventVenueTest.php

```diff
@@ -17,6 +17,8 @@ class EventVenueTest extends RdfKernelTestBase {
   public static $modules = [
     'oe_content',
     'oe_content_event',
+    'entity_reference_revisions',
+    'inline_entity_form',
```

Comment:
No, but we need these since the field storage is using entity reference revisions and the inline entity for widget. So to install event module we need these module to be present for config install.

Reply:
Ok. Fair enough.